tests/child_chain/test_child_chain.py

```diff
@@ -1,143 +1,89 @@
-from unittest.mock import Mock
-
 import pytest
-import rlp
-from ethereum import utils as u
-from plasma.child_chain.block import Block
-from plasma.child_chain.child_chain import ChildChain
-from plasma.child_chain.exceptions import (InvalidBlockMerkleException,
-                                           InvalidBlockSignatureException,
+from plasma.child_chain.exceptions import (InvalidBlockSignatureException,
                                            InvalidTxSignatureException,
                                            TxAlreadySpentException)
-from plasma.child_chain.transaction import Transaction
-
-AUTHORITY = b'\xfd\x02\xec\xeeby~u\xd8k\xcf\xf1d.\xb0\x84J\xfb(\xc7'
-tx_key = u.normalize_key(b'8b76243a95f959bf101248474e6bdacdedc8ad995d287c24616a41bd51642965')
-invalid_tx_key = u.normalize_key(b'8a76243a95f959bf101248474e6bdacdedc8ad995d287c24616a41bd51642965')
-block_key = u.normalize_key(b'3bb369fecdc16b93b99514d8ed9c2e87c5824cf4a6a98d2e8e91b7dd0c063304')
-invalid_block_key = u.normalize_key(b'3ab369fecdc16b93b99514d8ed9c2e87c5824cf4a6a98d2e8e91b7dd0c063304')
-newowner1 = b'\x8cT\xa4\xa0\x17\x9f$\x80\x1fI\xf92-\xab<\x87\xeb\x19L\x9b'
-amount1 = 200
-amount2 = 400
-
 
-@pytest.fixture
-def child_chain():
-    child_chain = ChildChain(AUTHORITY, Mock())
 
-    # Create some valid transations
-    tx1 = Transaction(0, 0, 0, 0, 0, 0, newowner1, amount1, b'\x00' * 20, 0, 0)
-    tx2 = Transaction(0, 0, 0, 0, 0, 0, newowner1, amount1, b'\x00' * 20, 0, 0)
+def test_apply_deposit(test_lang):
+    test_lang_string = '''
+        Deposit1[Owner1,100]
+    '''
 
-    # Create a block with those transactions
-    child_chain.blocks[1] = Block([tx1, tx2])
+    test_lang.parse(test_lang_string)
+    deposit_block_number = 1
+    deposit_block = test_lang.child_chain.blocks[deposit_block_number]
+    assert len(deposit_block.transaction_set) == 1
 
-    return child_chain
 
+def test_send_tx_with_sig(test_lang):
+    test_lang_string = '''
+        Deposit1[Owner1,100]
+        Transfer1[Deposit1,Owner2,100,Owner1,null,null,null,null]
```

Comment:
What do you think about having `null` be implied?

Reply:
Yeah, I think implied `null` is better. There are some cases where we can't have the implied `null`, but in most cases we can.